### ISSUE TITLE
assuming... no mac?

### ISSUE THREAD

**Issue by dumhed** [Norbyte/ositools#105]
So,.. I play mostly on PC... but at work... i have a mac... kinda want to play at work. :)

One of the mods I play with requires this dll... am I correct in assuming that this is not designed for the MacOS version?

**Comment by keventao** [comment]
yes, its a native macOS binary that completely different from the Windows binaries. 
-,-, it requires a rework of this whole project.
